pkg/cli/mirror/list/releases.go

```diff
@@ -22,6 +28,13 @@ type ReleasesOptions struct {
 	Version  string
 }
 
+type ReleaseVersion struct {
+	Major int
+	Minor int
+}
+
+const OCP_Release_Repo = "quay.io/openshift-release-dev/ocp-release"
```

Comment:
Nit: Go convention is to use camel case.

Reply:
Thanks, used to consts being all caps.